doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,458 @@
+
+    Do you wish the installer to prepend the Miniconda3 install location
+    to PATH in your /home/user/.bashrc ? [yes|no]
+    [no] >>> yes
+
+After opening a new terminal (for the changed ``PATH`` variable to take effect), you can run
+
+.. code-block:: bash
+
+   $ conda update --all
+
+to make sure you have the latest versions of all packages.
+
+Optionally, create a new conda environment to work with ODL.
+This is a very convenient way to have several "ecosystems" of Python packages in parallel without mutual interference:
+
+.. code-block:: bash
+
+    $ conda create --name odl-py35 python=3.5
+
+Follow the displayed instructions to enter/exit the newly created conda environment with name ``odl-py35``.
+See the `Managing conda environments`_ documentation for further information.
+
+If you use `Spyder`_ as integrated development environment (IDE, see `Development enviroment`_), you should also install it in the new conda environment and run it from there.
+Otherwise, Spyder is not able to use the packages in the conda environment:
+
+.. code-block:: bash
+
+    $ conda install spyder
+
+
+Installing ODL and its dependencies
+===================================
+Install ODL and all its (minimal) dependencies by running
+
+.. code-block:: bash
+
+    $ conda install -c odlgroup odl
+
+.. note::
+    To skip the ``-c odlgroup`` option in the future, you can permanently add the ``odlgroup`` conda channel (see `Managing conda channels`_):
+
+    .. code-block:: bash
+
+        $ conda config --append channels odlgroup
+
+    After that, ``conda install odl`` and ``conda update odl`` work without the ``-c`` option.
+
+Extra dependencies
+------------------
+The following packages are optional and extend the functionality of ODL.
+
+- Image and plot displaying capabilities:
+
+  .. code-block:: bash
+
+    $ conda install matplotlib
+
+- Faster FFT back-end using FFTW (currently not in mainstream conda):
+
+  * Install the `FFTW`_ C library version 3 (all possible precisions).
+    Use your Linux package manager for this task or consult the `Windows <http://fftw.org/install/windows.html>`_ or `MacOS <fftw.org/install/mac.html>`_ instructions, respectively.
+
+  * Run
+
+    .. code-block:: bash
+
+        $ pip install pyfftw
+
+- Wavelet transforms (currently not in mainstream conda):
+
+  .. code-block:: bash
+
+    $ pip install pywavelets
+
+- Simple backend for ray transforms:
+
+  .. code-block:: bash
+
+    $ conda install scikit-image
+
+- Bindings to the `ProxImaL`_ convex optimization package, an extension of `CVXPY`_:
+
+  .. code-block:: bash
+
+    $ pip install proximal
+
+- To run unit tests:
+
+  .. code-block:: bash
+
+    $ conda install pytest
+
+
+--------
+
+.. _alternative_2:
+
+*************************************************
+Alternative 2: Installing a release using ``pip``
+*************************************************
+
+TL;DR
+=====
+- Install `pip`_
+- Install `FFTW`_
```

Comment:
Does this really need to be in the TLDR? I find it very un-important since it is only an optimization that rather advanced users could take advantage of, numpy has the same functionality.